java-squid/src/main/java/org/sonar/java/resolve/AnnotationInstance.java

```diff
@@ -37,6 +37,10 @@ public void addValue(AnnotationValue annotationValue) {
     values.add(annotationValue);
   }
 
+  public Symbol.TypeSymbol getTypeSymbol() {
```

Comment:
@merciesa not sure this method is actually useful, for now requires at least an @visibleForTesting